test/distributed/tensor/parallel/test_ring_attention.py

```diff
@@ -0,0 +1,77 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates
```

Comment:
can we add this test to `_tensor` instead of tensor parallel?

Reply:
done